src/Dijkstra/AST/Examples/Maybe/Bind.agda

```diff
@@ -0,0 +1,106 @@
+open import Data.Nat renaming (ℕ to Nat)
+open import Dijkstra.AST.Core
+open import Dijkstra.AST.Maybe
+open        MaybeSyntax
+open        ASTOpSem         MaybeOpSem
```

Comment:
curious : have you tried deleting the rest of the imports one-by-one to see which are really needed?

Reply:
This issue will be superseded by another PR I plan on doing soon, which streamlines the user experience for the framework.